client/src/calendar.jsx

```diff
@@ -1,35 +1,95 @@
-/* eslint-disable class-methods-use-this */
 import React from 'react';
 import moment from 'moment';
+import styled from 'styled-components';
+
+const WeekdayHeader = styled.th`
+  font-size: smaller;
+  font-family: Roboto, sans-serif;
+  text-align: center;
+  color: rgb(117, 117, 117);
+  padding-left: 5px;
+  padding-right: 5px;
+`;
 
 class Calendar extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      dateContext: moment(),
       today: moment(),
     };
-    this.getHeader = this.getHeader.bind(this);
+    this.createHeader = this.getHeader.bind(this);
     this.weekdaysShort = moment.weekdaysShort();
     this.months = moment.months();
   }
 
-  getHeader() {
+  getYear() {
+    const { dateContext } = this.state;
+    return dateContext.format('Y');
+  }
+
+  getMonth() {
+    const { dateContext } = this.state;
+    return dateContext.format('MMMM');
+  }
+
+  getDaysInMonth() {
+    const { dateContext } = this.state;
+    return dateContext.daysInMonth();
+  }
+
+  getCurrentDate() {
+    const { dateContext } = this.state;
+    return dateContext.get('date');
+  }
+
+  getCurrentDay() {
```

Comment:
Probably could factor out this object destructuring to make it more DRY

Reply:
airbnb style guide enforces object destructuring, so I have to D: